email_repair.gemspec

```diff
@@ -14,7 +14,7 @@ Gem::Specification.new do |spec|
   spec.license       = 'MIT'
 
   spec.files         = Dir['{lib}/**/*', 'LICENSE', 'README.md']
-  spec.executables   = spec.files.grep(/^bin\//) { |f| File.basename(f) }
+  spec.executables   = spec.files.grep(%r{^bin\/}) { |f| File.basename(f) }
```

Comment:
This only requires `%r` syntax because there is a slash within the regex itself. The default is still `//`